#!/usr/bin/env python
PACKAGE = "camera"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("Kp_p", double_t, 0, "Kp_p", 2.8, 0, 15)
gen.add("Kd_p", double_t, 0, "Kd_p", 60, 0, 100)
gen.add("Kp_r", double_t, 0, "Kp_r", 2.8, 0, 15)
gen.add("Kd_r", double_t, 0, "Kd_r", 60, 0, 100)
gen.add("Kw_r", double_t, 0, "Kw_r", 0.2, 0, 0.5)
gen.add("Kw_p", double_t, 0, "Kw_p", 0.2, 0, 0.5)
gen.add("Kw_r_wall", double_t, 0, "Kw_r", 0,0,0.5)
gen.add("Kw_p_wall", double_t, 0, "Kw_p", 0,0,0.5)
#gen.add("pulseLength",int_t,0,"pulseLength",2,0,4);
gen.add("maxAngle", double_t, 0, "maxAngle", 10, 0, 15)

exit(gen.generate(PACKAGE, "FinalAttitudePublisher", "PIDFinal"))
